platform/qemu.go

```diff
@@ -15,17 +15,28 @@
 package platform
 
 import (
-	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
 	"strings"
+	"syscall"
 
 	"github.com/coreos/mantle/system/exec"
 )
 
+// boardSupportsFwCfg if the target system supports injecting
+// Ignition via the qemu -fw_cfg option.
+func boardSupportsFwCfg(board string) bool {
```

Comment:
Any reason to have this & `baseQemuArgs` be external functions rather than a class methods?

Reply:
I think of methods as taking (in Rust terms) a `&mut self` - and really the same as in C++/Python/Java, where an instance method normally either reads or writes something from the struct.

But this code just operates on a string, not a builder instance.

Reply:
Reworked to be a method.